capstone/src/main/webapp/script.js

```diff
@@ -12,44 +12,105 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Call the perspective API
- */
-async function callPerspective(text, lang) {
+const ATTRIBUTES_BY_LANGUAGE = {
+  'en': ['TOXICITY', 'SEVERE_TOXICITY', 'TOXICITY_FAST', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT', 'SEXUALLY_EXPLICIT', 'FLIRTATION'],
+  'es': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK_EXPERIMENTAL', 'INSULT_EXPERIMENTAL', 'PROFANITY_EXPERIMENTAL', 'THREAT_EXPERIMENTAL'],
+  'fr': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK_EXPERIMENTAL', 'INSULT_EXPERIMENTAL', 'PROFANITY_EXPERIMENTAL', 'THREAT_EXPERIMENTAL'],
+  'de': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT'],
+  'it': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT'],
+  'pt': ['TOXICITY', 'SEVERE_TOXICITY', 'IDENTITY_ATTACK', 'INSULT', 'PROFANITY', 'THREAT']
+};
+
+/** Collects the user's input and calls Perspective on it */
+async function submitInput() {
+  const textElement = document.getElementById('textForAnalysis');
+  if (!textElement) {
+    return;
+  }
+  const langElement = document.getElementById('languageForAnalysis');
+  if (!langElement) {
+    return;
+  }
+  const attributes = document.getElementById("available-attributes").getElementsByTagName('input');
+  const requestedAttributes = []
+  for (let attribute of attributes) {
+    if (attribute.checked == true) {
+      requestedAttributes.push(attribute.value);	
+    }	
+  }
+  await callPerspective(textElement.value, langElement.value, requestedAttributes);
+}
+
+/** Calls the perspective API */
+async function callPerspective(text, lang, requestedAttributes) {
   const response = await fetch('/call_perspective', {
```

Comment:
nit: it seems like a 'tab" is used as indention here as opposed to two spaces

Reply:
Fixed